caffe2/core/context_gpu.cu

```diff
@@ -136,6 +136,11 @@ static void Caffe2InitializeCuda() {
   // Restore the current device.
   CUDA_ENFORCE(cudaSetDevice(init_device));
 
+  RegisterTypeCallFunction(
```

Comment:
We need to do the same to Tensor<CPUContext>? I think RegisterShapeCallFunction for TensorCPU is done in tensor.cc